libs/common/src/lib/services/log.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core'
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LogService {
+  constructor() {}
+  log(message: string): void {
+    console.log(message)
+  }
+  warn(message: string): void {
+    console.warn(message)
+  }
+  error(message: string): void {
+    console.error(message)
```

Comment:
I don't think these shout output to the console eventually; like I said earlier, in the context of web components it would be super messy to log things to the console IMO; this service can be redirected to show notifications on the screen instead eventually I guess.

Reply:
Yes that's the purpose of the log service, we can change the implementation if required.
No opinion about console logging yet.